tests/test-utils.js

```diff
@@ -0,0 +1,81 @@
+import path from 'path';
+import webpack from 'webpack';
+import Promise from 'bluebird';
```

Comment:
Please no extra lib for `{Promise}` as every supported target has native `{Promise}` support 
`promisify()` can easily be polyfilled by a custom helper or a lightweight lib || `utils.promisify()`

Reply:
See comment below & I agree with you entirely. In fact I asked Sean about that shortly after looking at the example. 

`utils.promisify()` iirc isn't available in all our supported Node versions so if we were going to use this setup ( and i'm not ) i was thinking about something like pify.